src/box/xrow_update_field.h

```diff
@@ -263,6 +263,28 @@ struct xrow_update_op {
 	bool is_for_root;
 };
 
+/**
+ * Return weather operation is scalar (all other then insert/delete/set).
+ */
+static inline bool
+xrow_update_op_is_scalar(struct xrow_update_op *op)
+{
+	switch (op->opcode) {
+	case '+':
+	case '-':
+	case '&':
+	case '|':
+	case '^':
+	case ':':
+		return true;
+	case '=':
+	case '#':
+	case '!':
```

Comment:
These are not needed - they can be included into the `default`.

Reply:
I'd like instead to put `unreachable()` to default so we have all operation explicitly in the switch.